nio/src/main/scala/zio/nio/file/Files.scala

```diff
@@ -74,6 +81,13 @@ object Files {
     effectBlocking(Path.fromJava(JFiles.createTempFile(prefix.orNull, suffix, fileAttributes.toSeq: _*)))
       .refineToOrDie[IOException]
 
+  def createTempFileManaged(
+    suffix: String = ".tmp",
+    prefix: Option[String],
+    fileAttributes: Iterable[FileAttribute[_]]
+  ): ZManaged[Blocking, IOException, Path] =
+    ZManaged.make(createTempFile(suffix, prefix, fileAttributes))(release = deleteIfExists(_).ignore)
```

Comment:
Not sure what exactly should be used in the finalization block: ignore or orDie?
In most cases, it won't fail so I choose to ignore

Reply:
Yeah I'm not 100% sure either, but `ignore` seems good.